ci/default.nix

```diff
@@ -42,6 +42,22 @@ let
 
         programs.actionlint.enable = true;
 
+        programs.biome = {
+          enable = true;
+          settings.formatter = {
+            useEditorconfig = true;
+          };
+          settings.javascript.formatter = {
+            quoteStyle = "single";
+            semicolons = "asNeeded";
+          };
+          settings.json.formatter.enabled = false;
+        };
+        settings.formatter.biome.excludes = [
+          "*.min.js"
+          "pkgs/*"
```

Comment:
Interesting that this apparently doesn't need `pkgs/**`?

Makes me wonder if this is _actually_ matching any part of the path (effectively `**/pkgs/*/**`, with the leading `**/` and trailing `/**` added implicitly by treefmt)?

If so, I'd expect `pkgs` to work without any `/*` needed.

cc @jfly @brianmcgee